src/api/app/models/scm_webhook.rb

```diff
@@ -44,6 +44,18 @@ def pull_request_event?
     github_pull_request? || gitlab_merge_request?
   end
 
+  def not_allowed_pull_request_or_merge_request_event?
+    not_allowed_pull_request_event? || not_allowed_merge_request_event?
+  end
+
+  def not_allowed_pull_request_event?
+    github_pull_request? && ScmWebhookEventValidator::NOT_ALLOWED_PULL_REQUEST_ACTIONS.include?(@payload[:action])
+  end
+
+  def not_allowed_merge_request_event?
+    gitlab_merge_request? && ScmWebhookEventValidator::NOT_ALLOWED_MERGE_REQUEST_ACTIONS.include?(@payload[:action])
+  end
+
```

Comment:
I wonder if these methods can be converted to positive, using ALLOWED_*_ACTIONS instead of NOT_ALLOWED_*_ACTIONS :thinking: